test/Libraries/DynamoPythonTests/PythonEditTests.cs

```diff
@@ -195,10 +266,10 @@ public void VerifyPythonLoadsFromCore()
             // reference to specific testing nodes in test graph
             string[] testingNodeGUIDS = new string[]
             {
-                "845d532fdf874d939f2ed66509413ea6",
-                "cb037a9debd54ce79a4007b6ea11de25",
-                "a9bb1b12fbbd4aa19299f0d30c9f99b2",
-                "b6bd3049034f488a9bed0373f05fd021"
+                "845d532f-df87-4d93-9f2e-d66509413ea6",
```

Comment:
@QilongTang This test was not asserting the truth values as it doesn't match the GUID pattern. Had to insert the '-' character.

Reply:
@reddyashish Thanks for fixing! Make sense